internal/cmd/billing/command.go

```diff
@@ -0,0 +1,23 @@
+package billing
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+)
+
+type command struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+func New(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "billing",
+		Short:       "Manage Confluent Cloud billing.",
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireCloudLogin},
```

Comment:
We haven't implemented API key support for endpoints like yours yet. Will do soon!
```suggestion
		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonApiKeyCloudLogin},
```